python/paddle/fluid/tests/unittests/test_requantize_mkldnn_op.py

```diff
@@ -0,0 +1,106 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+def format_reorder(out, size):
+    in_n = size[0]
+    out_h = size[2]
+    out_w = size[3]
+    out_c = size[1]
+    out_tmp = np.zeros((in_n, out_h, out_w, out_c))
+    for n in range(in_n):
+        for i in range(out_h):
+            for j in range(out_w):
+                for m in range(out_c):
+                    out_tmp[n, i, j, m] = out[n, m, i, j]
+    return out_tmp.reshape(in_n, out_c, out_h, out_w)
```

Comment:
duplicated with 
https://github.com/PaddlePaddle/Paddle/blob/fe8f28c957a1cf44bbb92cf7752de3cfbc6c7cc8/python/paddle/fluid/tests/unittests/test_conv2d_int8_mkldnn_op.py#L32-L43

Reply:
Yes, this function is same as test_conv2d_int8_mkldnn_op, but I think it's not a good choice to do "import function from conv2d" in requantize op test, because there isn't relationship between these two ops. What do you think about it? How about we move this function to a public file, like "op_test" ?

Reply:
> How about we move this function to a public file, like "op_test"?

I think a public file is OK, but not `op_test`. @sfraczek @xiaolil1

Reply:
@luotao1 May i add these code to mkldnn_op_test.py ?

Reply:
> May i add these code to mkldnn_op_test.py

That's OK @xiaolil1